src/hotspot/cpu/aarch64/aarch64_neon.ad

```diff
@@ -271,46 +258,49 @@ instruct vcvt2Lto2I(vecD dst, vecX src)
   ins_pipe(pipe_class_default);
 %}
 
-instruct vcvt4Bto4I(vecX dst, vecD src)
```

Comment:
4BtoX should be supported as the min_vector_size() for byte type is 4.

Reply:
OK, I'll add rules for 4B.